scripts/install_third_party.sh

```diff
@@ -148,6 +148,24 @@ if [ ! -d "$TOOLS_DIR/isort-4.2.15" ]; then
   rm isort-4.2.15.tar.gz
 fi
 
+# Install GitPython and its dependencies.
```

Comment:
A couple of points here -- make this as three separate `echo`s instead of one `Installing GitPython and its dependencies` so that even if any one of them is already installed, we skip that.